arclightning_frontend/games.html

```diff
@@ -53,6 +73,62 @@ <h1>Arc Lightning</h1>
 
 	]
             },
+
+	methods: {
+		sendSelectedGame: function() {
+			//check to see if game exists
+			var gameSelected = this.items[this.selectedIndex].exe_path
```

Comment:
i wouldn't use exe_path. Once you refactor the data to look like the above comment, just post using the unique key id